tests/functional/adapter/basic/test_base_mat.py

```diff
@@ -14,13 +14,14 @@
     run_dbt,
     check_result_nodes_by_name,
 )
+from tests.functional.adapter.utils.test_utils import DATALAKE
 
-
+# Unable to insert variable into docstring, so "rav-test" is hardcoded
 schema_base_yml = """
 version: 2
 sources:
   - name: raw
-    database: "rav-test"
+    database: "rav-test" 
```

Comment:
should this not be DATALAKE as well?

Reply:
I tried to insert a variable, but it turns out inserting variables into docstrings isn't trivial, especially when the "{" notation is also used to insert variables from the dbt config.